projects/kit/components/prompt/prompt.style.less

```diff
@@ -1,18 +1,19 @@
 @import 'taiga-ui-local';
 
 .t-content:not(:empty) {
-    margin-bottom: 1.25rem;
+    margin-bottom: 0.875rem;
 }
 
 .t-buttons {
     display: flex;
     justify-content: flex-end;
-    margin-top: 0.75rem;
-    width: 100%;
+    flex-wrap: wrap;
```

Comment:
Could you add Cypress screenshot test for this case?

https://taiga-ui--pr4443-prompt-orwo26dc.web.app/components/prompt/API?no=Very%20Long%20long%20text&yes=Long%20long%20text